db/migrate/20211022131607_drop_cockpit.rb

```diff
@@ -0,0 +1,35 @@
+class DropCockpit < ActiveRecord::Migration[6.0]
+  class AssignedServerRole < ActiveRecord::Base; end
+
+  class ServerRole < ActiveRecord::Base; end
+
+  class SettingsChange < ActiveRecord::Base; end
+
+  class MiqUserRole < ActiveRecord::Base
+    has_and_belongs_to_many :miq_product_features,
+                            :join_table => "miq_roles_features",
+                            :class_name => "DropCockpit::MiqProductFeature"
```

Comment:
Is this being used?  If not, can we drop it?

Reply:
it was used in the tests. can approach from the other end. not as easy but it works

Reply:
If it's only for the tests, then it should only live in the tests, however, that may be moot as discussed in https://github.com/ManageIQ/manageiq-schema/pull/613#discussion_r744948932

Reply:
gone